### PR TITLE
Better styled components experience

### PR DESCRIPTION
**Pull request by Drache93**
https://www.styled-components.com/docs/tooling#babel-plugin

Thoughts?

### CODE REVIEW

File: packages/react-scripts/config/webpack.config.js [hunk 1]
```diff
@@ -383,6 +383,7 @@ function buildConfig(webpackEnv) {
         require.resolve('babel-preset-react-app'),
       ],
       plugins: [
+        require.resolve("babel-plugin-styled-components"),
```
Comment (miraage) [inline packages/react-scripts/config/webpack.config.js]: Use single quotes

**Comment by vteremasov** [comment]
I know guys struggled with this unreadable classnames

**Comment by Drache93** [comment]
@miraage Any other babel/script extras worth adding while we're here?

**Comment by vteremasov** [comment]
Not babel stuff but I think adding tsconfig `baseurl` can make imports look much better.

**Comment by miraage** [comment]
@vteremasov this pull is about `react-scripts` package. Do you mean to update the TS template?

**Comment by vteremasov** [comment]
> @vteremasov this pull is about `react-scripts` package. Do you mean to update the TS template?

Yes, but tbh I am not sure it's not already there

**Pull request merged**